neo/rawio/intanrawio.py

```diff
@@ -210,15 +210,31 @@ def _parse_header(self):
         discontinuous_timestamps = np.diff(timestamp) != 1
         timestamps_are_not_contiguous = np.any(discontinuous_timestamps)
         if timestamps_are_not_contiguous:
+            # Mark a flag that can be checked after parsing the header to see if the timestamps are continuous or not
             self.discontinuous_timestamps = True
             if not self.ignore_integrity_checks:
                 error_msg = (
-                    "Timestamps are not continuous, this could be due to a corrupted file or an inappropriate file merge. "
-                    "Initialize the reader with `ignore_integrity_checks=True` to ignore this error and open the file. \n"
-                    f"Timestamps around discontinuities: {timestamp[discontinuous_timestamps]}"
+                    "Timestamps are not continuous, likely due to a corrupted file or inappropriate file merge.\n"
+                    "To open the file anyway, initialize the reader with `ignore_integrity_checks=True`.\n\n"
+                    "Discontinuities Found:\n"
+                    "+-----------------+-----------------+-----------------+-----------------------+\n"  
+                    "| Discontinuity   | Previous        | Next            | Time Difference       |\n"
+                    "| Index           | (Frames)        | (Frames)        | (Seconds)             |\n"
+                    "+-----------------+-----------------+-----------------+-----------------------+\n"
                 )
-                raise NeoReadWriteError(error_msg)
 
+                amplifier_sampling_rate = self._global_info["sampling_rate"]
+                for idx in np.where(discontinuous_timestamps)[0]:
```

Comment:
Could we do a better than idx :) 

maybe like bad_timestamp_index or something more verbose? That way if we go back and read this we have a clearer idea of what this was.

Reply:
Yes. You are right.

Reply:
Done.
I  also coalesced all the logic into a private function to not break the main sequence as this became too large.